include/fastad_bits/mat.hpp

```diff
@@ -0,0 +1,96 @@
+#pragma once
```

Comment:
Change all function definitions from this form
```cpp
void f() { /* do stuff */}
```
to
```cpp
void f()
{
/* do stuff */
}
```
I know the code looks a little longer, but this really helps with readability in the long run